README.md

```diff
@@ -1,3 +1,27 @@
-# JaneliaDataStandards
-A central hub for Janelia's bioimaging schemas, standards, and data formats.
-This project is at a very preliminary stage.
+# Janelia Data Standards
+
+## Purpose
+
+The Janelia Data Standards group was formed by bioimaging developers who have encountered specific, practical bioimaging data dilemmas that international community standards don’t yet cover. When cutting-edge experiments produce new kinds of data or metadata, developers may be forced to modify existing file formats, or occasionally create new ones. For example, it is often unclear how to store and describe high dimensional images, image tiles that form parts of a whole, or images accompanied by large ancillary data sets. Even though developers may be making reasonable choices on their own, a chaotic landscape can result if they do not communicate their choices, and encourage others to store similar data in a similar way.
```

Comment:
```suggestion
The Janelia Data Standards group was formed by bioimaging developers who have encountered specific, practical bioimaging data dilemmas for which there is little or no guidance from international community standards. When cutting-edge experiments produce new kinds of data or metadata, developers may be forced to modify existing file formats, or occasionally create new ones. For example, it is often unclear how to store and describe high dimensional images, image tiles that form parts of a whole, or images accompanied by large ancillary data sets. Even though developers may be making reasonable choices on their own, a chaotic landscape can result if they do not communicate their choices, and encourage others to store similar data in a similar way.
```

Reply:
Sometimes I wonder if we are just repeating what came before in esoteric places such as the Confocal Listserv:
https://lists.umn.edu/cgi-bin/wa?A0=CONFOCALMICROSCOPY

Reply:
I feel like this is pretty different from that project, which seems like part Q&A forum, part announcements, and isn't curated. Should I add something more to the README to clarify what makes us different? Maybe contrast this project with forums like image.sc?